src/components/dashboard/Dashboard.js

```diff
@@ -251,6 +247,17 @@ const styles = StyleSheet.create({
     padding: 8,
     height: '256px',
   },
+  bigNumberStyles: {
+    fontFamily: 'Roboto-Bold',
+    fontSize: normalize(36),
+    fontWeight: '700',
+    marginRight: normalize(3),
+  },
+  bigNumberUnitStyles: {
+    fontFamily: 'Roboto-Bold',
```

Comment:
`RobotSlab-Bold` is stated in wireframes

Reply:
Changed it on the dashboard's main big number.